src/main/java/org/variantsync/diffdetective/diff/difftree/NodeType.java

```diff
@@ -15,10 +15,7 @@ public enum NodeType {
     ELIF("elif"),
 
     // Artefact types
-    ARTEFACT("code"),
-
-    // Extra type for the root
-    ROOT("ROOT");
+    ARTEFACT("code");
```

Comment:
Rename this to artifact? What would be affected?

Reply:
The following node formats use `NodeType.ARTIFACT.name` for output (`toLabel`):
- DebugMiningDiffNodeFormat
- RWCompositePatternNodeFormat
- DebugDiffNodeFormat
- MappingsDiffNodeFormat
- FormulasAndLineNumbersNodeFormat
- TypeDiffNodeFormat

It's also used by `DiffNode.toString` and
`DiffNodeLabelPrettyfier.prettyPrintTypeAndMapping`.

There is one case where `NodeType.ARTIFACT.name` is used for reading data:
`DebugMiningDiffNodeFormat.fromEncodedTypes` (using `NodeType.fromName`).

Given the *debug* in the name `DebugMiningDiffNodeFormat` I changed the value of
`NodeType.ARTIFACT.name` to "artifact".

Reply:
As long as the rename is consistent for write _and_ read operations (which should be the case as every such operation should rely on these constans) everything should be fine. We will lose the ability to parse old evaluation data but that is not necessary.